doc/cookbook/performance.md

```diff
@@ -0,0 +1,192 @@
+# Performance
+
+- [Performance](#performance)
+  - [Basic: Load Balance](#basic-load-balance)
+  - [Performance: Compression and Caching](#performance-compression-and-caching)
+    - [Compression in filter `Proxy`](#compression-in-filter-proxy)
+    - [Caching in filter `Proxy`](#caching-in-filter-proxy)
+    - [Caching in `HTTPServer`](#caching-in-httpserver)
+  - [References](#references)
+    - [Proxy Compression](#proxy-compression)
+    - [Proxy Caching](#proxy-caching)
+    - [HTTPServer route rule caching](#httpserver-route-rule-caching)
+
+Easegress supports compression and caching in HTTPPipeline and HTTPServer for improving performance purposes as a revert proxy.
+
+
+## Basic: Load Balance
+
+```yaml
+name: pipeline-reverse-proxy
+kind: HTTPPipeline
+flow:
+  - filter: proxy
+filters:
+  - name: proxy
+    kind: Proxy
+    mainPool:
+      servers:
+      - url: http://127.0.0.1:9095
+      - url: http://127.0.0.1:9096
+      - url: http://127.0.0.1:9097
+      loadBalance:
+        policy: roundRobin
+```
+
+## Performance: Compression and Caching
+
+### Compression in filter `Proxy`
+
+* Easegress proxy filter supports `gzip` type compression. It can save the brandwidth between client and Easegress and reduce the time cost.
```

Comment:
```suggestion
* Easegress proxy filter supports `gzip` type compression. It can save the bandwidth between the client and Easegress and reduce the time cost.
```